README-zh.md

```diff
@@ -799,7 +800,7 @@ Awesome Mac
 * [Streaker](https://github.com/jamieweavis/streaker) - GitHub贡献和统计跟踪菜单栏应用程序。 [![Open-Source Software][OSS Icon] ![Freeware][Freeware Icon]](https://github.com/jamieweavis/streaker)
 * [The Unarchiver](https://theunarchiver.com/) - 解压许多不同种类的归档压缩文件。 [![Open-Source Software][OSS Icon]](https://bitbucket.org/kosovan/theunarchiver) ![Freeware][Freeware Icon]
 * [Timing](https://timingapp.com/) - Mac 的自动时间和生产力跟踪。
-* [Unarchive One](https://cleanerone.trendmicro.com/unarchiver-one/) - 快速解压单个多个不同种类的压缩文件/压缩文件到各类场景压缩格式。 ![Freeware][Freeware Icon] [![App Store][app-store Icon]](https://apps.apple.com/cn/app/unarchiver-one-rar-zip-tool/id1127253508?l=en&mt=12)
+* [Unarchive One](https://cleanerone.trendmicro.com/unarchiver-one/?utm_source=github&utm_medium=referral&utm_campaign=githubproject) - 快速解压单个多个不同种类的压缩文件/压缩文件到各类常见压缩格式。 ![Freeware][Freeware Icon] [![App Store][app-store Icon]](https://apps.apple.com/app/apple-store/id1127253508?pt=444218&ct=GitHub&mt=8)
```

Comment:
https://apps.apple.com/app/apple-store/id1127253508?pt=444218&ct=GitHub&mt=8
Can't open in browser

Reply:
@jaywcjlove U.S. region can be opened

Reply:
in China => `https://apps.apple.com/cn/app/`

```diff
- https://apps.apple.com/app/
+ https://apps.apple.com/cn/app/
```

Reply:
@jaywcjlove Hi, for some reason, the link provided here should be a foreign link, is this okay? Or is it possible to provide two links here, one for domestic and one for foreign countries

Reply:
`README-zh.md` -> Chinese users. `https://apps.apple.com/cn/app/` Can be opened in a browser

Reply:
@jaywcjlove Ok i am going to change it now

Reply:
@jaywcjlove Ok I adjust it. Thanks